catchup/wsFetcher.go

```diff
@@ -168,3 +163,45 @@ func (w *wsFetcherClient) Close() error {
 	w.pendingCtxs = make(map[context.Context]context.CancelFunc)
 	return nil
 }
+
+// RequestBlock send a request for block <round> and wait until it receives a response or a context expires.
+func (w *wsFetcherClient) RequestBlock(ctx context.Context, round basics.Round) (rpcs.WsGetBlockOut, error) {
```

Comment:
`RequestBlock` -> `requestBlock`

Reply:
Please review the refactored output of this function and the usage. It looks like you've eliminated few error cases, so make sure to take advantage of that.